openshift-ci/build-root/Dockerfile

```diff
@@ -2,4 +2,6 @@
 
 FROM registry.ci.openshift.org/openshift/release:golang-1.17
 
-RUN yum -y install make wget gcc git httpd-tools
+RUN yum -y install make wget gcc git httpd-tools && \
+    mkdir -p $(shell go env GOPATH)/bin && \
+    curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell go env GOPATH)/bin v1.37.0
```

Comment:
Do we really want to use the `install.sh` from `master` each time? Can we use the one from `v.1.37.0` release instead to avoid surprises?